mp_api/client/mprester.py

```diff
@@ -60,19 +60,12 @@
 from mp_api.client.routes.materials.materials import MaterialsRester
 from mp_api.client.routes.molecules import MoleculeRester
 
-_DEPRECATION_WARNING = (
```

Comment:
Need confirm on this deletion, I didn't see this warning being accessed.

Reply:
Should be fine